docs/user-guide/running.md

```diff
@@ -15,6 +15,50 @@ When your environment is running, you can [deploy your workload](deploying-workl
 
 To force device re-registration remove EdgeDevice CR from the cluster and `/var/local/yggdrasil/device` directory from the device
 
+---
+## Automatically
+
+### Prerequisites
+- Clone all Flotta components: [flotta-operator](https://github.com/project-flotta/flotta-operator), [flotta-device-worker](https://github.com/project-flotta/flotta-device-worker) and [yggdrasil](https://github.com/RedHatInsights/yggdrasil).
+- On minikube: add to `/etc/hosts`: `<minikube ip> <host name>`.
```

Comment:
WDYT to remember to add execution privilege for hack/start-*.sh scripts?

Reply:
shouldn't the files have the correct permissions in github?